crates/matrix-sdk-ui/src/timeline/event_item/content/mod.rs

```diff
@@ -198,6 +198,7 @@ impl TimelineItemContent {
                 let reactions = Default::default();
                 let thread_root = None;
                 let in_reply_to = None;
+                let thread_summary = None;
```

Comment:
Probably the latest preview item should also return a MsgLikeKind, not a MsgLike, then. (Future refactor :innocent:)